src/hearings/containers/request-hearing/hearing-timing/hearing-timing.component.ts

```diff
@@ -361,7 +361,8 @@ export class HearingTimingComponent extends RequestHearingPageFlow implements On
         ...this.hearingRequestMainModel.hearingDetails,
         duration,
         hearingWindow,
-        hearingPriorityType: this.priorityForm.value.priority
+        hearingPriorityType: this.priorityForm.value.priority,
+        hearingType: this.priorityForm.value.specificDate
```

Comment:
hearingType is not specificDate.

Reply:
this is fixed logic implemented using the dates values.